flashcards/backend/comments/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+
+from comments.models import Comment
+
+
+class CommentAdminConfig(admin.ModelAdmin):
+    """Class **CommentAdminConfig** displays comments list in admin panel
+    based on :class:`comments.models.Comment` model."""
+
+    list_display = (
+        "user",
+        "flashcard",
+        "deck",
+        "comment",
+        "created_at",
+    )
+    search_fields = (
+        "comment",
+    )
+    list_display_links = ("user",)
+    save_on_top = True
+    list_filter = (
+        "user",
+        "flashcard",
+        "deck",
+    )
+    list_per_page = 50
```

Comment:
Having this kind of list filter isn't very handy 😅 I risk to say that It would be more problematic. Instead I would add `user__email`, `deck__name`, `flashcard__name` in `search_fields`. This opinion applys to all your admin configurations. 

<img width="342" alt="Screenshot 2023-04-25 at 16 56 16" src="https://user-images.githubusercontent.com/92261595/234317814-3bae766f-1a8e-4adf-891e-53eee9283061.png">
<img width="307" alt="Screenshot 2023-04-25 at 16 56 47" src="https://user-images.githubusercontent.com/92261595/234317846-4367d4a4-5a04-40d7-b001-b41368b7c40f.png">